x/incentives/keeper/suite_test.go

```diff
@@ -41,8 +41,6 @@ func (suite *KeeperTestSuite) setupAddr(addrNum int, prefix string, balance sdk.
 		prefixBz := make([]byte, 8)
 		_, _ = rand.Read(prefixBz)
 		prefix = string(prefixBz)
-	} else {
-		prefix = fmt.Sprintf("%8.8s", prefix)
```

Comment:
I removed this because the width and precision format specifier options for %s are based on unicode runes, not bytes. So essentially it was causing a strange bug where it would append to a certain []byte, bumping it from 8 bytes to 9 bytes to get it up to 8 runes. Then, the assert for lock creation would [fail](https://github.com/osmosis-labs/osmosis/runs/7176310153?check_suite_focus=true#step:6:56), because it would be a 21 byte address instead of 20.

If we do actually mean to make this exactly 8 unicode runes, I can change the assert statement when we create locks to check for rune length, not byte/string length.

More context: https://stackoverflow.com/a/60681952

Reply:
Thanks for fixing this! I'm pretty sure its some copy/paste issue from someone pulling something of stackoverflow and not looking into the details